packages/cli/src/commands/deployment/deploy.ts

```diff
@@ -68,6 +80,21 @@ export default class Deploy extends Command {
       }
     }
 
+    const queryAD = {
+      unsafe: flags.queryUnsafe,
+      subscription: flags.querySubscription,
+      queryTimeout: flags.queryTimeout,
+      maxConnection: flags.maxConnection,
+      Aggregate: flags.Aggregate,
+    };
+    const indexerAD = {
+      unsafe: flags.unsafe,
+      batchSize: flags.batchSize,
+      subscription: flags.indexerSubscription,
+      historicalData: flags.historicalData,
+      workers: flags.workers,
+    };
```

Comment:
Do we need to do any validation on these? Or is that all handled on the hosted service? I see the descriptions say value ranges

Reply:
I just tested it  in host service swagger, all the configurations will be verified on host service

Reply:
![Snip20221019_13](https://user-images.githubusercontent.com/92004774/196555099-f9b04b94-822a-45d6-82b3-ee3db8602322.png)